logbook-core/src/test/java/org/zalando/logbook/io/EnforceCoverageTest.java

```diff
@@ -22,15 +22,23 @@
 
 import com.google.gag.annotation.remark.Hack;
 import com.google.gag.annotation.remark.OhNoYouDidnt;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
 import org.junit.Test;
 
 @Hack
 @OhNoYouDidnt
 public final class EnforceCoverageTest {
 
     @Test
-    public void shouldUseMockHeadersConstructor() {
-        new MockHeaders();
+    public void shouldUseByteStreamsConstructor() {
+        new ByteStreams();
     }
 
+    @Test
+    public void shouldCoverByteStreams() throws IOException {
```

Comment:
Granted, but it's still a valid test beyond enforcing code coverage. Of course, one could always argue that the only reason we have tests in the first place is to satisfy the coverage requirement.